index.html

```diff
@@ -69,6 +69,149 @@ <h1 class="header_job">I’m a software developer</h1>
+                <ul class="portfolio_techs">
+                  <li>Ruby on rails</li>
+                  <li>css</li>
+                  <li>Javascript</li>
+                  <li>html</li>
+                </ul>
+                <button class="portfolio_button">See Project</button>
+              </div>
+            </div>
+            <div class="work_item">
+              <div class="portfolio_image">image</div>
+              <div class="portfolio_text">
+                <h2 class="work_title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="portfolio_techs">
+                  <li>Ruby on rails</li>
+                  <li>css</li>
+                  <li>Javascript</li>
+                  <li>html</li>
+                </ul>
+                <button class="portfolio_button">See Project</button>
+              </div>
+            </div>
+            <div class="work_item">
+              <div class="portfolio_image">image</div>
+              <div class="portfolio_text">
+                <h2 class="work_title">Multi-Post Stories Gain+Glory</h2>
+                <ul class="portfolio_techs">
+                  <li>Ruby on rails</li>
+                  <li>css</li>
+                  <li>Javascript</li>
+                  <li>html</li>
+                </ul>
+                <button class="portfolio_button">See Project</button>
+              </div>
+            </div>
+          </section>
+        </div>
+      </section>
+
+      <section id="about">
+        <section class="about_top">
+          <h2 class="about_title">About Me</h2>
+          <p class="about_text">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <button class="about_button">Get my Resume</button>
+        </section>
+        <section class="about_bottom">
+          <div class="languages">
+            <div class="about_icons">
+              <img src="/images/languages.png" alt="alt" class="" />
+            </div>
+            <h2>languages</h2>
+            <ul class="language_lists">
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </ul>
+          </div>
+          <div class="languages">
+            <div class="about_icons">
+              <img src="/images/frameworks.png" alt="alt" class="" />
+            </div>
+            <h2>Frameworks</h2>
+            <ul class="language_lists">
+              <li>Bootstrap</li>
+              <li>TailwindCSS</li>
+
+              <li>Rub on rails</li>
+              <li>Selenium</li>
+              <li>capybara</li>
+            </ul>
+          </div>
+          <div class="languages">
+            <div class="about_icons">
+              <img src="/images/skills.png" alt="alt" class="" />
```

Comment:
- **_[OPTIONAL]_** The about icons are not visible 🙈 and as mentioned by the previous reviewer, here you need to use a relative address when you apply the backgrounds 🏗. Please remember to add './' for every relative address. Use `<img src="./images/languages.png" alt="alt" class="" />` instead of `<img src="/images/languages.png" alt="alt" class="" />`

![image](https://user-images.githubusercontent.com/43172164/205279004-033a4f31-6fca-4750-ad73-ac3e2d3cfdf9.png)